src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj

```diff
@@ -38,7 +38,7 @@
   <ItemGroup>
     <PackageReference Include="ILRepack" Version="2.0.18" />
     <PackageReference Include="Irony" Version="1.1.0" />
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
Can we unify these version numbers like we do for NuGet and LibZipSharp? So we only need to change them in one place?

Reply:
Partly; I don't see how to "reasonably" get `PackagingTest.cs` to use `$(NewtonsoftJsonPackageVersion)`, but all the other uses have been updated.

Reply:
We might not necessarily need to keep `PackagingTest` updated. It's probably just as helpful to be testing old NuGets for compatibility as the newest ones.